### PR TITLE
[Feat] [Translations] [Uzbek]: uz/advanced/readme

### PR DESCRIPTION
**Pull request by mcpeblocker**

### CODE REVIEW

File: site/docs/uz/advanced/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Agar kodingiz juda murakkab bo'lib qolsa, [I Bo'lim](./structuring.md)ni o'qing.
+Agar juda ko'p xabarlar boshqaruvini yo'lga qo'yishingiz kerak bo'lsa, [II Bo'lim](./scaling.md)ni o'qing.
+Agar botingiz ishonchliliki haqida qayg'ursangiz, [III Bo'lim](./reliability.md)ni o'qing.
+Agar cheklovlarga duch kelayotgan bo'lsangiz (masalan: 429 xatoliklar), [IV Bo'lim](./flood.md)ni o'qing.
+
+Agar siz API so'rovlarga ishlov berishingiz kerak bo'lsa, grammY buni [transformer funksiyalar](./transformers.md) yordamida amalga oshirishingizni maslahat beradi.
+
+Bundan tashqari, grammY [proksi](./proxy.md) ishlatish imkonini ham beradi.
+
+Va nihoyat biz siz uchun botingizni ishga tushirayotganda [esingizda tutishingiz kerak bo'lgan ba'zi bir narsalarni](./deployment.md) tayyorlaganmiz.
```
Comment (sad1go0) [inline site/docs/uz/advanced/README.md:21]: narsalarni -> ma'lumotlarni

File: site/docs/uz/advanced/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Bu yerda hech qanday yangi narsa kiritilgan emas, bu shunchaki taxminiy muammolarning markazlashtirilgan holdagi jamlanmasi.
```
Comment (sad1go0) [inline site/docs/uz/advanced/README.md:22]: narsa -> ma'lumot

**Review by sad1go0 — CHANGES_REQUESTED** [review]
Perfect!

**Review by sad1go0 — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
You're making good progress! Keep up the good work—and remember to delete your branches when a PR is merged 😉

Optionally, you may want to check out the GitHub Refined browser extension which does this automatically for you (but only if you feel like using it).

**Pull request merged**